mapclassify/classifiers.py

```diff
@@ -258,11 +257,12 @@ def quantile(y, k=4):
             f"Not enough unique values in array to form {k} classes. "
             f"Setting k to {k_q}.",
             UserWarning,
+            stacklevel=1,
```

Comment:
```suggestion
            stacklevel=2,
```

stacklevel is never 1

Reply:
I think I must perpetually mess up `stacklevel`. It is in my DNA.